SaveGameEditor/GameSave.cs

```diff
@@ -257,11 +381,11 @@ public void ReadHeaderFromFile(string headerPath)
             // Read the date of the save
             for (var i = 0; i < Header.saveDate.Count; i++)
             {
-                SaveDate[i] = Header.saveDate[i]; // Need to test if it's possible to write to this dynamic array without an intermediate one
+                SaveDate[i] = Header.saveDate[i];
             }
         }
 
-        public void WriteSaveToFile(string savePath, dynamic saveJsonObject)
+        public void WriteSaveToFile(string savePath, dynamic saveJsonObject, bool isFarewell = false)
```

Comment:
I'd like to introduce some Enum
emun SaveType
{
  Regular,
  Bonus
}
And use it wherever it is necessary (if there are several places in code, which uses the same logic)